cafebabel/templates/users/detail.html

```diff
@@ -43,9 +43,29 @@ <h2>About</h2>
 </section>
 
 <section>
-  <h2>Contributions</h2>
-  {% for article in articles %}
-    {{ card(article) }}
-  {% endfor %}
+  {% if articles %}
+    <input type=radio name=contribution-type-switch id=articles-list checked>
+    <label for=articles-list>Articles</label>
+    <div class=contributions>
+    {% for article in articles %}
+      {{ card(article) }}
+    {% endfor %}
+    </div>
+  {% endif %}
+  {% if translations %}
+    <input type=radio name=contribution-type-switch id=translation-list {% if not articles %}checked{% endif %}>
+    <label for=translation-list>Translations</label>
+    <div class=contributions>
+      {% for translation in translations %}
+        {{ card(translation) }}
+      {% endfor %}
+    </div>
+  {% endif %}
+  {% if articles|length >= 25 or translations|length >= 25 %}
```

Comment:
I think we should swap this line with the one below for performances matter, this way we do not count all articles/translations when they are all displayed (which might be costly).

And the test should be `if articles|length == 25 or translations|length == 25` or is there a edge-case I forget?

Reply:
I don't get what you mean? @davidbgk Can you swap the line yourself please?
And I don't know how to fix the test either, where should I put this line: `if articles|length == 25 or translations|length == 25`?